utils/logging.py

```diff
@@ -0,0 +1,6 @@
+from loguru import logger
+
+
+class Logger:
+    logger.remove()
+    logger.add("debug.log", format="{time:DD:MM:YYYY:h:m:s} {level} {message}", level="DEBUG")
```

Comment:
I would suggest to create `.env` file and store some parameters like `LOG_LEVEL`, `BROWSER_NAME` etc